app/views/admin/link_check_reports/_form.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= form_tag link_check_reports_path(link_reportable: { type: "manual", manual_id: reportable.id }), remote: true do %>
```

Comment:
Does this form make sense with only a button inside it?

Reply:
I used a form for the button because it gets called three times inside https://github.com/alphagov/manuals-publisher/blob/832cf202d591e1f6d71527b1b77cc47d93a0adf5/app/views/admin/link_check_reports/_link_check_report.html.erb so I wanted to minimize code dulication